server/controllers/entryController.js

```diff
@@ -89,7 +89,38 @@ class EntryController {
         }
     }
 
-    static deleteEntry = (req, res) => {}
+    static deleteEntry = async (req, res) => {
+        try {
+            let {
+                entryId
+            } = req.params
+
+            const findEntryId = `SELECT * FROM mydiaryEntries WHERE entryid = $1`;
+            const { rows } = await pool.query(findEntryId, [entryId]);
+
+            if (!rows[0]) {
+                return res.status(400).send({
+                    status: 400,
+                    error: 'Entry entered is invalid'
```

Comment:
@Ntare22 You can change `Entry entered is invalid` to `Entry entered doesn't exist`